core/CompiledLevel.h

```diff
@@ -0,0 +1,19 @@
+#ifndef SORBET_CORE_COMPILED_LEVEL_H
+#define SORBET_CORE_COMPILED_LEVEL_H
+
+namespace sorbet::core {
+enum class CompiledLevel {
+
+    // This file has no compiled sigil present. The behavior here is the same as for `# compiled: false`, but it's
+    // useful to distinguish files that have no sigil present for stats.
+    Unknown = 0,
+
+    // This file is explicitly opted-out of compilation.
+    False = 1,
+
+    // This file should be compiled.
+    True = 1,
```

Comment:
You probably want this to be `2`?

Reply:
I sure do. That's what I get for adding `None` after the others :)